composer.json

```diff
@@ -8,7 +8,7 @@
   "require": {
     "php": ">=5.3.0",
     "psr/log": "^1.0.0",
-    "barracudanetworks/forkdaemon-php": "^1.0.10"
+    "barracudanetworks/forkdaemon-php": "^1.1"
```

Comment:
This should be `^1.1.1` so we don't pull in non php 7.2 compliant version 1.1.0

Reply:
I'm actually not sure if we want to change this. You technically have support for the later versions here.  We keep more support leaving it the way it is